README.md

```diff
@@ -7,3 +7,21 @@ This is the official OmniAuth strategy for authenticating to Clearbit.
     use OmniAuth::Builder do
       provider :clearbit, ENV['CLEARBIT_KEY'], ENV['CLEARBIT_SECRET']
     end
+
+## Advanced Usage
+
+To use this strategy against local, dev, or staging environments you'll need to
+override the values in that are [hardcoded for production](lib/omniauth/strategies/clearbit.rb#L4:L8) for the appropriate environment.
+
+In dev, this might look like
+
+    use OmniAuth::Builder do
+        provider :clearbit,
+          ENV['CLEARBIT_KEY'],
+          ENV['CLEARBIT_CONSUMER_SECRET'],
+          client_options: {
```

Comment:
lets use example w/ ENV?

https://github.com/clearbit/clearbit-x/blob/bfa7c469c5160d76a6ba880b48072d2cc3f37bad/app/routes/auth.rb#L5-L13

Reply:
yeah, i think i just wanted to highlight that the value should actually be different. but ill make a note of that